### PR TITLE
download bulk management history if showBulkManagement

### PR DESCRIPTION
**Pull request by muselesscreator**
**TL;DR -**

Download Bulk Grades history if showBulkManagement (vs if has masters track)

JIRA: [AU-77](https://openedx.atlassian.net/browse/AU-77)

**Developer Checklist**
- [x] Test suites passing
- [x] Documentation and test plan updated, if applicable
- [ ] Received code-owner approving review
- [ ] Bumped version number [package.json](../package.json)

**Testing Instructions**

[ How should a reviewer test this PR? ]

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [x] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

File: src/data/thunkActions/roles.js [hunk 1]
```diff
@@ -28,6 +28,9 @@ export const fetchRoles = () => (
           dispatch(fetchTracks());
           dispatch(fetchCohorts());
           dispatch(fetchAssignmentTypes());
+          if (selectors.root.showBulkManagement(getState())) {
```
Comment (nsprenkle) [inline src/data/thunkActions/roles.js:31]: Running locally, this is returning `undefined`.

**Review by nsprenkle — CHANGES_REQUESTED** [review]
Isn't working locally.

**Review by nsprenkle — APPROVED** [review]
Beautiful :shipit:
